repo/fsrepo/migrations/ipfsfetcher_test.go

```diff
@@ -0,0 +1,47 @@
+package migrations
+
+import (
+	"bufio"
+	"context"
+	"testing"
+)
+
+func TestIpfsFetcher(t *testing.T) {
+	t.Skip("manually-run dev test only")
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	fetcher := NewIpfsFetcher("", 0, nil)
+	defer fetcher.Close()
+
+	rc, err := fetcher.Fetch(ctx, "go-ipfs/versions")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer rc.Close()
+
+	var out []string
```

Comment:
I think this would be clearer if called `lines` :)

Reply:
done